src/components/layout/layout.js

```diff
@@ -1,17 +1,25 @@
-import React from 'react';
+import React, { Suspense } from 'react';
```

Comment:
Suspense is currently causing issues with the build due to [compatibility issues with Gatsby](https://sung.codes/blog/2019/07/31/react-suspense-doesnt-work-with-gatsby-yet/)